src/main/resources/META-INF/services/net.minecraftforge.fml.loading.moddiscovery.IModLocator

```diff
@@ -1,2 +1,3 @@
 net.minecraftforge.fml.loading.moddiscovery.ModsFolderLocator
-net.minecraftforge.fml.loading.moddiscovery.ExplodedDirectoryLocator
\ No newline at end of file
+net.minecraftforge.fml.loading.moddiscovery.ExplodedDirectoryLocator
+net.minecraftforge.fml.loading.moddiscovery.ClasspathLocator
```

Comment:
Does this screw up runtime when this service provider is missing?

Reply:
If it was missing, I would guess so, that's why I wrote it so that it disables itself (refuses to scan) outside dev.
The alternative would be to make the userdev injection capable of injecting entries into the service locator file, and then both the locator and this line could be moved into the userdev sourceset and would be 100% missing outside dev.